modules/tensor_mechanics/doc/content/source/kernels/lagrangian/HomogenizedTotalLagrangianStressDivergence.md

```diff
@@ -0,0 +1,32 @@
+# HomogenizedTotalLagrangianStressDivergence
+
+!syntax description /Kernels/HomogenizedTotalLagrangianStressDivergence
+
+## Overview
+
+This object provides the total Lagrangian stress equilibrium and 
+Jacobian.  It is identical to the [`TotalLagrangianStressDivergence`](TotalLagrangianStressDivergence.md)
```

Comment:
It seems like the first sentence should say something like 'for the homogenization system'

Reply:
Yep.